style.css

```diff
@@ -29,7 +29,75 @@ article {
 	margin-bottom: 10px;
 	background-color: aqua;
 	width: 50%;
-	height: 150px;
+	height: 600px;
```

Comment:
we can have the same box for  nice UI as height :500px

Reply:
i did push the box with the size of 500px. Don't get why it is 600px?